ee/localserver/krypto-ec-middleware_test.go

```diff
@@ -209,6 +209,139 @@ func TestKryptoEcMiddleware(t *testing.T) {
 	}
 }
 
+func Test_AllowedOrigin(t *testing.T) {
+	t.Parallel()
+
+	counterpartyKey, err := echelper.GenerateEcdsaKey()
+	require.NoError(t, err)
+
+	challengeId := []byte(ulid.New())
+	challengeData := []byte(ulid.New())
+
+	var tests = []struct {
+		name           string
+		requestOrigin  string
+		allowedOrigins []string
+		logStr         string
+		expectedStatus int
+	}{
+		{
+			name:           "no allowed specified",
+			requestOrigin:  "auth.example.com",
```

Comment:
The origin will include the scheme, so I think either a) logic should be adjusted to handle URLs with and without schemes, or b) the test cases should use origins with schemes.

Reply:
good call. I added schemes.